koku/masu/database/ocp_report_db_accessor.py

```diff
@@ -534,6 +534,52 @@ def get_distinct_pvcs(self, start_date, end_date, cluster_id):
             )
             return [(pvc[0], pvc[1], pvc[2]) for pvc in unique_pvcs]
 
+    def populate_platform_distributed_cost_sql(self, start_date, end_date, distribution, provider_uuid, rate_type):
+        """
+        Populate the platform cost distribution of a customer.
+
+        args:
+            start_date (datetime, str): The start_date to calculate monthly_cost.
+            end_date (datetime, str): The end_date to calculate monthly_cost.
+            distribution: Choice of monthly distribution ex. memory
+            provider_uuid (str): The str of the provider UUID
+        """
```

Comment:
Why do we need to pass in `rate_type` here? Is it ever _NOT_ going to be `metric_constants.DISTRIBUTE_PLATFORM_RATE_TYPE`?

Reply:
Good catch! I originally wrote this method to work for both platform & worker distribution, but then decided it would be easier to combine them if possible during the worker distribution PR. This is just a left over param from when I was playing with that idea. Let me get rid of that. 🙇